src/app/components/code-block/index.html

```diff
@@ -26,6 +26,13 @@
     >
       Passes a code sample to the component. This allows you to pass the code sample as a string so that you can provide it from a hosting component or JSON string. This property binds data, so you can use brackets to evaluate the template expression: <sky-code>[code]</sky-code>. Without the brackets, Angular treats the value as a string and initializes the target property with the string value instead of evaluating the data value. This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fileName"
+      isOptional="true"
+      defaultValue=""
+    >
+      Renders a filename on the codeblock.
```

Comment:
Where is the filename rendered? If it's in the code block header, should this be `Displays a filename in the code block's header.`?

If not and we leave the description as is, `code block` should be 2 words.